minimint-api/Cargo.toml

```diff
@@ -8,26 +8,27 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.58"
 async-trait = "0.1"
-bitcoin = { version = "0.28.1", features = [ "rand", "serde" ] }
+bitcoin = { version = "0.28.1", features = ["rand", "serde"] }
```

Comment:
What software is doing the reformatting here? I'd rather avoid that since it's not checked by CI and will thus diverge again (leading to autoreformatting once you touch it again).

Reply:
I think IntelliJ idea Settings>tools>actions on safe, I turned that on because I thought it would sort the dependencies but it dosn't do that and I forgot to turn it off

Reply:
Ah, there is another setting under Settings>Languages&Frameworks>Rust>Rustfmt "Run rustfmt on Save", that's what I have turned on and what CI verifies. That's better imo because it's independent of any particular IDE.